examples/secure-s3-bucket/main.tf

```diff
@@ -0,0 +1,12 @@
+provider "aws" {
+  version = "~> 2.0"
+  region  = var.aws_region
+}
+
+module "s3-bucket" {
+  source = "../.."
+
+  create = true
+  bucket = "mineiros-test-bucket"
+  acl    = "private"
```

Comment:
is default and can be omitted

Reply:
work in progress :) we should come up with a complete example

Reply:
the minimal should always work... complete is quiet impossible ;) too many different combinations ;)

Reply:
indeed, but we should provide a as close to complete example that covers most of this modules  feature as possible. I always fell that examples are the best ways of how to get started.

Reply:
yes... totally agree... so real world examples are the best.... cut&paste and go.. fire&forget ;)